0-ad-history-mod-package/simulation/templates/structures/spart/field.xml

```diff
@@ -3,6 +3,10 @@
   <Identity>
     <Civ>spart</Civ>
     <SpecificName>Agros</SpecificName>
+    <History>
+        Most of Greece is mountainous and not well suited for farming. Some think only twenty percent of Greece was usable for farming. Less of this was farm fields because Greece also cultivated grape vineyards and olive orchards. What was fields was mostly sown with barely, although wheat was used as well.
```

Comment:
```suggestion
        Most of Greece is mountainous and not well suited for farming. Some think only twenty percent of Greece was usable for farming. Less of this was farm fields because Greece also cultivated grape vineyards and olive orchards. What was fields was mostly sown with barley, although wheat was used as well.
```